templates/uui-nextjs-template-app-router/helpers/appContext.ts

```diff
@@ -0,0 +1,8 @@
+export async function getAppContext() {
```

Comment:
Let's remove this helper and usage since using of App context is not popular case